### ISSUE TITLE
capsule and maven-assembly-plugin 4.5+

### ISSUE THREAD

**Issue by atao60** [puniverse/capsule#93]
When I build a project with capsule 1.0.1 and maven-assembly-plugin 2.4.1, everything runs fine.

As soon as I upgrade maven-assembly-plugin to 2.5 (or above), I get:
```
$ java -Dcapsule.log=verbose -jar target/dateutils.jar 
CAPSULE: Jar: /home/pierre/devspace/samples/fatjar/mkyong-logging/dateUtils/target/dateutils.jar
CAPSULE: Platform: linux
CAPSULE EXCEPTION: Capsule manifest contains a Class-Path attribute. Use App-Class-Path=null and/or Dependencies=null instead.
java.lang.IllegalStateException: Capsule manifest contains a Class-Path attribute. Use App-Class-Path=null and/or Dependencies=null instead.
	at Capsule.validateManifest(Capsule.java:2758)
	at Capsule.finalizeCapsule0(Capsule.java:821)
	at Capsule.finalizeCapsule(Capsule.java:816)
	at Capsule.<init>(Capsule.java:739)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at Capsule.newCapsule0(Capsule.java:5501)
	at Capsule.newCapsule(Capsule.java:5493)
	at Capsule.myCapsule(Capsule.java:356)
	at Capsule.main0(Capsule.java:382)
	at Capsule.main(Capsule.java:374)
```
Did I miss something?

**Comment by pron** [comment]
Why is there a `Class-Path` attribute in the JAR file? What's its value? Is there a way of telling Maven-assembly not to generate it? 

BTW, Have you considered using the [Capsule Maven Plugin](https://github.com/chrischristo/capsule-maven-plugin)?

**Comment by atao60** [comment]
When I build the sample "https://github.com/puniverse/capsule-demo", I get the same result.
After your post, I dug a little more. In the pom of the capsule-demo, for the maven-assembly-plugin,  I removed from the configuration/archive/manifest section:
```
<classpathPrefix>lib</classpathPrefix>
<addClasspath>true</addClasspath>
```
Now everything runs fine for maven-assembly-plugin versions from 2.4 to 2.5.5 but it failled with the 2.6: no output at all!

About  Capsule Maven Plugin: yes indeed, but I'm a slow guy. I move step by step and first play a little with the sample provided with capsule.

**Comment by pron** [comment]
> but it failled with the 2.6: no output at all!

Try running with logs: `-Dcapsule.log=verbose`
Also, compare the manifest with that of the older versions.

**Comment by atao60** [comment]
It seems there is an issue with v. 2.6 of maven-assembly-plugin.
I started with:
``` 
mvn clean package
java -Dcapsule.log=verbose -jar target/foo-1.0-capsule-fat.jar
```
Everything runs fine.
Without waiting, I started again with the same commands: the new fat-jar doesn't throw any output.

I can't find any differences between the two jar files! Sizes are the same and contents seem also. But something must be different as one fatjar works and not the other.
